Tests/Shared.Specs/XDocumentAssertionSpecs.cs

```diff
@@ -659,6 +659,21 @@ public void When_asserting_an_xml_document_is_equivalent_to_different_xml_docume
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void when_an_xml_element_has_attributes_and_is_self_closing_it_should_succeed()
+        {
+            // Arrange
+            XDocument actual = XDocument.Parse("<xml><a attr=\"x\"/><b id=\"foo\"/></xml>");
+            XDocument expected = XDocument.Parse("<xml><a attr=\"x\"/><b id=\"bar\"/></xml>");
+
+            // Act
```

Comment:
Maybe we should start using this "light" AAA format everywhere

Reply:
Pure muscle memory, cause the the light AAA format is what I use at work.
I'll push an update to this PR to use the other format for consistency.

But I'll be happy to create a another PR converting to light AAA syntax. I don't recall having seen this heavier syntax anywhere else.

Reply:
I introduced the heavy format myself years ago, but I don't see a lot of value in it anymore.

Reply:
+1 on the 'light' AAA syntax. I find the current format over the top, and many times the comment to code ratio is very high, making it extra hard to navigate around the tests.